src/OpenTelemetry.Exporter.Jaeger/JaegerExporterHelperExtensions.cs

```diff
@@ -38,7 +38,16 @@ public static TracerProviderBuilder AddJaegerExporter(this TracerProviderBuilder
                 throw new ArgumentNullException(nameof(builder));
             }
 
-            var exporterOptions = new JaegerExporterOptions();
+            JaegerExporterOptions exporterOptions;
+            if (builder is IResolvingTracerProviderBuilder resolvingTracerProviderBuilder)
```

Comment:
Here is an example of how something like JaegerExporter can use IResolvingTracerProviderBuilder to inject options.

Reply:
Reminded me of this https://github.com/census-instrumentation/opencensus-python/issues/394#issuecomment-482772410.

Reply:
@reyang Is that a good thing or a bad thing? I'm hoping since it was your comment it is a good thing 🤣

Reply:
It is a very good thing, most of the customers don't want to learn how the engine works, they just want to drive the car.
These configuration is the steering wheel.